test/unit/application_test.rb

```diff
@@ -86,6 +86,13 @@ class ApplicationTest < ActiveSupport::TestCase
       assert_equal false, application.on_aws?
     end
 
+    should "default to not be in deploy freeze" do
+      @atts.delete :on_aws
```

Comment:
```suggestion

```

This is also irrelevant for the previous test, as it's not in `@atts`.

Reply:
Ah good catch!!! Removed from the other tests so it's consistent